### PR TITLE
[FluidDynamicsApplication] Adding fluid adjoint steady test

### PR DESCRIPTION
**Pull request by sunethwarna**
This adds a steady adjoint sensitivity test case.

### CODE REVIEW

File: applications/FluidDynamicsApplication/tests/adjoint_vms_sensitivity_2d.py [hunk 1]
```diff
@@ -188,6 +188,26 @@ def testCylinder(self):
...
+            Sensitivity[0].append(test._GetSolver().main_model_part.GetNode(1968).GetSolutionStepValue(SHAPE_SENSITIVITY_Y))
+
+            # calculate sensitivity by finite difference
+            step_size = 0.00000001
+            FDSensitivity = self._computeFiniteDifferenceDragSensitivity([1968],step_size,'./AdjointVMSSensitivity2DTest/steady_cylinder_test',[1.0,0.0,0.0],'./AdjointVMSSensitivity2DTest/steady_cylinder_test.dat')
+            self.assertAlmostEqual(Sensitivity[0][0], FDSensitivity[0][0], 4)
+            self.assertAlmostEqual(Sensitivity[0][1], FDSensitivity[0][1], 2)
+            self._removeH5Files("MainModelPart")
+            self._removeFile("./AdjointVMSSensitivity2DTest/steady_cylinder_test.dat")
+            self._removeFile("./steady_cylinder_test.time")
```
Comment (philbucher) [inline applications/FluidDynamicsApplication/tests/adjoint_vms_sensitivity_2d.py]: there is a function for this in kratos_utilities

Comment (sunethwarna) [reply applications/FluidDynamicsApplication/tests/adjoint_vms_sensitivity_2d.py]: thanks :+1: , done :)

**Comment by philbucher** [comment]
:+1:

File: applications/FluidDynamicsApplication/tests/AdjointVMSSensitivity2DTest/steady_cylinder_test_adjoint_parameters.json [hunk 2]
```diff
@@ -0,0 +1,138 @@
+{
+    "test_settings" : {
+        "execute_solve" : true
```
Comment (msandre) [inline applications/FluidDynamicsApplication/tests/AdjointVMSSensitivity2DTest/steady_cylinder_test_adjoint_parameters.json]: Do we still need this setting?

File: applications/FluidDynamicsApplication/tests/AdjointVMSSensitivity2DTest/steady_cylinder_test_parameters.json [hunk 3]
```diff
@@ -0,0 +1,160 @@
+{
+    "test_settings" : {
+        "execute_solve" : true
```
Comment (msandre) [inline applications/FluidDynamicsApplication/tests/AdjointVMSSensitivity2DTest/steady_cylinder_test_parameters.json]: Same here.

**Review by msandre — COMMENTED** [review]
I will approve after checking the settings.

**Comment by sunethwarna** [comment]
@msandre thanks :), I made the changes :+1:

**Review by msandre — APPROVED** [review]
Thanks

**Pull request merged**